test/misc.jl

```diff
@@ -1239,3 +1239,67 @@ end
     @test (@allocations "a") == 0
     @test (@allocations "a" * "b") == 1
 end
+
+@testset "sleep precision" begin
+    println("GREP TARGET 1729 A")
```

Comment:
I wonder if adding functionality for a logging blurb when a testset starts to run and ends to the basic testset directly would be good :thinking: